lib/helpers/getDomNodeProfile.js

```diff
@@ -26,7 +26,8 @@ export function getDomNodeProfile(el, options = {}) {
     selector: getDomNodePath(el),
     tag_name: el.tagName,
     text: el.text,
-    text_content: getTextContent(el, options),
+    cursor:  document.defaultView.getComputedStyle(el).getPropertyValue('cursor'),
```

Comment:
nit: unless there's a specific reason for it to be here, can we keep the list alphabetized?

Reply:
This is probably fine, but I don't think `getComputedStyle` is the most performant method. In the future, maybe we can come up with some way to track how long it's taking events to send to make sure we're not inadvertently creating bottlenecks. If we had short term concerns we could do something like:
```
let cursor = el.style.cursor;
if ( ! cursor ) {
  cursor = window.getComputedStyle(el).getPropertyValue('cursor');
}​
```

Reply:
Also fwiw, it sounds like you no longer need to do `.getPropertyValue('cursor')` you can just call `.cursor`, and it also sounds like you can call `getComputedStyle` on `window` - not sure if that has performance implications, so do what you will with that knowledge